bowling/player.hpp

```diff
@@ -1,3 +1,13 @@
 #pragma once
+#include <vector>
 
-class Player {};
+class Player {
+public:
+    int getScore() const;
+    void setScore(int score);
+
+    void countScore(std::vector<std::pair<int, int>> score);

```

Comment:
for me int countScore(...) would better solution

Reply:
Ok, I was working on your proposition and it did not return anything, but we can change this of course ;)